### ISSUE TITLE
stuck on reloading in Android TV

### ISSUE THREAD

**Issue by Kira-0007** [itszechs/DriveStream#121]
I have checked wifi speed.. its very good and everything working fine but as in one drive i have many files and when i scroll down a little its keep reloading and stuck on it.. as result I can't access below videos...
Please do something about it... 🙏🏻
![IMG_20230806_220100](https://github.com/itszechs/DriveStream/assets/105588608/c484dc28-0252-4fab-892f-9b347d9aed88)

**Comment by itszechs** [comment]
A quick solution would be to ***star*** the files you want.

**Comment by Kira-0007** [comment]
Yeah this can be a quick solution but it will only works if we are using application with same id in mobile too...
Otherwise if we are using only in tv then starred won't help..

If there is any permanent solution for this please try to fix it.. 🙏🏻

**Comment by itszechs** [comment]
I've an idea on what's going wrong, but if you could record a short video demonstrating the problem, I will be able to confirm. Thanks!

**Comment by Kira-0007** [comment]
https://github.com/itszechs/DriveStream/assets/105588608/bc24743e-c5d4-4094-96c4-d156761019a6

Here bro..

**Comment by Kira-0007** [comment]
I was going to upload video at first but then i thought I don't know if that is allowed or not due to copyright concern

**Comment by Kira-0007** [comment]
Heyy bro.. did you find any solution for it?

**Comment by Kira-0007** [comment]
Hello bro... Found any solution of it yet ??

**Comment by Kira-0007** [comment]
Heyy bro i know you must be busy on something and i don't want to disturb you.. but this problem i mentioned above is still a big problem.. i hope you can fix this please

**Comment by itszechs** [comment]
I was not able to reproduce this issue. I will get back to you for more information when I get around to fixing this.

**Comment by Kira-0007** [comment]
Yes bro please.. i am always available to help.. i thought there may be some problem on Android TV then i tried on Fire TV Stick but issue was same..

For myself i can starred any file as a temporary solution but for my friends or family members if i login to TV then Starr option couldn't help for them..

I will always be in your debt.. please bro find some solution of it.. hope you are understanding 🙏🏻♥️

**Comment by itszechs** [comment]
@Kira-0007 can you share the file/folder list which is in the video from Google Drive website

**Comment by Kira-0007** [comment]
Yes sure.. but for safety purpose can we talk in private chat where i can send you file/folders whatever you need... If it is ok for you

**Comment by itszechs** [comment]
> Yes sure.. but for safety purpose can we talk in private chat where i can send you file/folders whatever you need... If it is ok for you

Sure, you can reach me out on discord {redacted}

**Comment by Kira-0007** [comment]
Friend request sent.. please check

**Comment by Varun23102002** [comment]
Hey bro u got solution to this error in android tv of loading

**Comment by sagya210** [comment]
Working for me smoothly in android tv.. No issue whats so ever... But quality somehow 

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/03a05a9e-9af6-47cb-8ed5-76b2bd94b553" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/3d255c22-507a-476d-8729-c5a55f2d45f0" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/83e8f916-2505-4be3-a465-c6c1ba3baa42" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/c28a3f58-16dd-44ef-a3c0-e0c6362cde90" />

downgrade.. [Kindly]([url](url)) add option for external video app like vlc..

**Comment by sagya210** [comment]
Running smoothly on android tv, android phone.. No issue so far.. Make app more android tv friendly.. Currently using Bluetooth keyboard & mouse app for handling app in tv.. Content quality somehow little bit downgrade.. 1080P content played smoothly.. 4K content lagging.. Please add option for external player like vlc.. It will be great.. HATS OFF FOR THE WORK.. THANKS..

**Comment by sagya210** [comment]
https://github.com/user-attachments/assets/e25f9efc-31ba-4d9d-bc3b-8a92bda8d070

4k content lag issue

**Comment by itszechs** [comment]
> 2025-08-26-13-22-19.mp4 
> 4k content lag issue

is it the same way if you play downloaded file on [mpv-android](https://github.com/mpv-android/mpv-android)
